cmd/shield/main.go

```diff
@@ -1817,15 +1830,27 @@ func main() {
 				return err
 			}
 
-			if err := RestoreArchive(id, string(b)); err != nil {
+			var taskUUID string = ""
+			if taskUUID, err = RestoreArchive(id, string(b)); err != nil {
```

Comment:
likewise here, drop the combined if-initializer-condition and just do the assignment outright.